### PR TITLE
Postgres in express template

### PR DESCRIPTION
**Pull request by FrancescoXX**
- Rename "item" routes to "local-items"
- Rename "db-item" routes to "dynamodb-item"
- Create routes for "postgresql-item". Postgres db url, port, and name are passed in via env variables.
- For all GET paths, return full list of items unless there's a query parameter "id" passed in the url - then only return that item

Added .dockerignore file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,9 +96,9 @@ By default the server starts on port 80. You can change this to port 8000 by def
 const PORT = process.env.STAGE === "local" ? 8000 : 80;
 ```
 
-#### Adding an Item in Memory
+#### Adding a local Item in Memory
```
Comment (zsimjee) [inline README.md:99]: I think it was fine before

File: README.md [hunk 2]
```diff
@@ -95,9 +96,9 @@ By default the server starts on port 80. You can change this to port 8000 by def
...
 
-To add an item in memory, call the `/item` endpoint with an HTTP PUT verb. This can be done on Unix/MacOS systems using cUrl: 
+To add a local item in memory, call the `/local-item` endpoint with an HTTP PUT verb. This can be done on Unix/MacOS systems using cUrl:
```
Comment (zsimjee) [inline README.md:101]: To add an item in memory…

File: src/config/postgres.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { Pool } from 'pg';
+
+const pool = new Pool({
+  host: process.env.PG_HOST || 'localhost',
+  port: parseInt(process.env.PG_PORT || '5433'),
```
Comment (zsimjee) [inline src/config/postgres.ts:5]: Is the default RDS port 5433 or 5432? Can we have it match the default?

**Review by zsimjee — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (FrancescoXX) [reply README.md:100]: ok done

File: README.md (hunk 2 above)
Comment (FrancescoXX) [reply README.md:102]: done

File: src/config/postgres.ts (hunk 3 above)
Comment (FrancescoXX) [reply src/config/postgres.ts:5]: yes the RDS default is 5432, so let's change it

File: README.md [hunk 4]
```diff
@@ -97,7 +99,7 @@ const PORT = process.env.STAGE === "local" ? 8000 : 80;
 
 #### Adding an Item in Memory
 
-To add an item in memory, call the `/item` endpoint with an HTTP PUT verb. This can be done on Unix/MacOS systems using cUrl: 
+To add a an item in memory, call the `/local-item` endpoint with an HTTP PUT verb. This can be done on Unix/MacOS systems using cUrl:
```
Comment (zsimjee) [inline README.md:102]: keep this as “an”

File: src/postgresdb-item.ts [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+    try {
+      const client = await pool.connect();
+      const toRet = await client.query('SELECT * from items WHERE id = $1', [req.params.id]);
+      client.release();
+      return res.status(200).json(toRet.rows[0]);
+    } catch (error) {
+      return res.status(500).json('Error on getPostgresDbItem' + error);
+    }
+  }
+}
```
Comment (zsimjee) [inline src/postgresdb-item.ts:37]: let’s restructure this code block. Some psuedo-code:

```
try {
const client = await pool.connect();
let queryResult;
if (!req.params.id) {
  queryResult = await (client.query(…)).rows;
} else {
  queryResult = await (client.query(…)).rows[0];
}

client.release();
return res.status(200).json(queryResult);
} catch…
```

**Review by zsimjee — APPROVED** [review]

**Pull request merged**